Index: config/gen/makefiles/dynpmc.in
===================================================================
--- config/gen/makefiles/dynpmc.in	(revision 19313)
+++ config/gen/makefiles/dynpmc.in	(working copy)
@@ -13,6 +13,7 @@
 # There is no config probe for libquantum yet, so add 'quantumreg' manually if desired
 #
 PMCS = \
+  lgp \
   dynlexpad \
   foo \
   rotest \
Index: src/call_list.txt
===================================================================
--- src/call_list.txt	(revision 19313)
+++ src/call_list.txt	(working copy)
@@ -430,6 +430,6 @@
 # win32
 l lttl
 
-i pip
-i piS
-S i
+# dynpmc/lgp.pmc
+v JOII
+I JOII
