Index: config/gen/call_list/misc.in
===================================================================
--- config/gen/call_list/misc.in	(revision 29697)
+++ config/gen/call_list/misc.in	(working copy)
@@ -353,3 +353,7 @@
 i    JOt
 i    Jt
 i    Ji
+
+# Used by parrot-lgp ( dynpmc/lgp.pmc, http://pr.perl6.cz/parrot-lgp )
+v    JOII
+I    JOII
Index: config/gen/makefiles/dynpmc.in
===================================================================
--- config/gen/makefiles/dynpmc.in	(revision 29697)
+++ config/gen/makefiles/dynpmc.in	(working copy)
@@ -11,6 +11,7 @@
 # gdbmhash is only built when 'libgdbm.so' is available
 #
 PMCS = \
+  lgp \
   dynlexpad \
   foo \
   rotest \
